notebooks/MAST/m_dwarf_flares_hst/m_dwarf_flares_hst.ipynb

```diff
@@ -0,0 +1,1412 @@
+{
```

Comment:
<p>As the reader I was wondering, why not just search for all HST images of the target by name? And then I realized, oh, we're searching by coordinate, and the coordinate in RA and Dec changes significantly. If I search by name, would MAST not find all the right observations because it translates a name to a J2000 coordinate and only searches that coordinate? By how much does Proxima Centauri actually move in arcseconds during this time period? (That could be an exercise for the reader.) </p><p> </p><p>Some more of this could be explained in the introduction. When you said "Downloading and extracting these data can therefore be non-trivial," in the intro, that's where you could expand upon the problem if you don't take the proper motion into account.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>